plf-tomcat-resources/src/main/resources/bin/setenv.sh

```diff
@@ -133,7 +134,15 @@ fi
 CATALINA_OPTS="$CATALINA_OPTS -Dexo.jcr.session.tracking.active=${EXO_JCR_SESSION_TRACKING}"
 
 # JVM memory allocation pool parameters
-CATALINA_OPTS="$CATALINA_OPTS -Xms${EXO_JVM_SIZE_MIN} -Xmx${EXO_JVM_SIZE_MAX} -XX:MaxPermSize=${EXO_JVM_PERMSIZE_MAX}"
+CATALINA_OPTS="$CATALINA_OPTS -Xms${EXO_JVM_SIZE_MIN} -Xmx${EXO_JVM_SIZE_MAX}"
+
+# PLF-6510: Configure the JVM according to the version
+cmd=$(java -jar $CATALINA_HOME/bin/exo-tools.jar isJava8OrSuperior)
+if [ $? == 0 ]; then
+  CATALINA_OPTS="$CATALINA_OPTS -XX:MaxMetaspaceSize=${EXO_JVM_METASPACE_SIZE_MAX}"
```

Comment:
Why do we want to enforce a MetaSpace size limit ? (by default, the MetaSpace is dynamically resize)